draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -514,26 +579,39 @@ irrelevant fields omitted for ease of reading).
 ~~~http
 Signature:
  sig1;
-  sig=*MEUCIQ...;
+  sig=*MEUCIQ...*;
+  ...
+  validity-url="https://example.com/resource.validity.1511157180";
+  cert-url="https://example.com/oldcerts";
+  date=1511128380; expires=1511733180,
+ sig2;
+  sig=*MEQCIG...*;
+  ...
+  validity-url="https://example.com/resource.validity.1511157180";
+  cert-url="https://example.com/newcerts";
+  date=1511128380; expires=1511733180,
+ thirdpartysig;
+  sig=*MEYCIQ...*;
   ...
-  validityUrl="https://example.com/resource.validity.1511157180";
-  certUrl="https://example.com/oldcerts";
-  date=1511128380; expires=1511733180
+  validity-url="https://thirdparty.example.com/resource.validity.1511161860";
+  cert-url="https://thirdparty.example.com/certs";
+  date=1511478660; expires=1511824260
 ~~~
 
-At 2017-11-27 11:02 UTC, `sig1` has expired, so the client needs to fetch
-`https://example.com/resource.validity.1511157180` (the `validityUrl` of `sig1`)
-to update that signatures. This URL might contain:
+At 2017-11-27 11:02 UTC, `sig1` and `sig2` have expired, but `thirdpartysig`
+doesn't exipire until 23:11 that night, so the client needs to fetch
+`https://example.com/resource.validity.1511157180` (the `validity-url` of `sig1`
+and `sig2`) to update those signatures. This URL might contain:
```

Comment:
s/to/if it wishes to/? (i.e. A client may use `thirdpartysig` without updating the others if it validates and meets its needs?)